README.md

```diff
@@ -152,25 +144,29 @@ a) z głosem stanowiącym – członkowie zwyczajni,
 
 b) z głosem doradczym – członkowie wspierający oraz zaproszeni goście.
 
-Walne Zebranie Członków może być zwyczajne i nadzwyczajne.
+Walne Zebranie Członków jest zwoływane w trybie zwyczajnym lub nadzwyczajnym. Zebranie odbywa się w siedzibie Stowarzyszenia lub zostaje przeprowadzone z wykorzystaniem środków komunikacji elektronicznej. Zarząd powiadamia członków Stowarzyszenia o terminie zebrania, planowanym porządku obrad oraz:
 
-Dla zwołania Walnego Zebrania Członków podaje się dwa terminy. Jeśli w pierwszym terminie nie zbierze się kworum, Walne Zebranie Członków odbywa się w drugim terminie. Członków Stowarzyszenia zawiadamia się o terminach Walnego Zebrania za pośrednictwem poczty elektronicznej.
+a) jego szczegółowej lokalizacji – w przypadku zebrania przeprowadzanego w siedzibie Stowarzyszenia,
 
-Aby Walne Zebranie Członków mogło się rozpocząć w pierwszym terminie wymagane jest kworum, czyli obecność co najmniej połowy ogólnej liczby zwyczajnych członków Stowarzyszenia. W drugim terminie kworum nie jest wymagane.
+b) sposobie dołączenia do spotkania – w przypadku zebrania przeprowadzanego z wykorzystaniem środków komunikacji elektronicznej
 
-Oba terminy Walnego Zebrania Członków nie mogą być odległe bardziej niż 14 dni kalendarzowych.
+za pośrednictwem poczty elektronicznej, w terminie uzależnionym od trybu organizacji zebrania.
 
-Walne Zgromadzenie Członków zwyczajne jest zwoływane nie rzadziej niż raz na rok przez Zarząd Stowarzyszenia w siedzibie stowarzyszenia. Termin obrad Zarząd podaje do wiadomości wszystkich członków co najmniej 14 dni przed pierwszym terminem zebrania.
+Walne Zebranie Członków w trybie zwyczajnym zwoływane jest przez Zarząd Stowarzyszenia nie rzadziej, niż raz na rok. Zarząd powiadamia członków Stowarzyszenia o zebraniu z wyprzedzeniem co najmniej 14 dni kalendarzowych. Poczas obrad Walnego Zebrania Członków zwołanego w trybie zwyczajnym nie jest wymagane kworum.
 
-Walne Zebranie Członków nadzwyczajne może się odbyć w każdym czasie w siedzibie stowarzyszenia. 
-
-Nadzwyczajne Walne Zebranie Członków zwoływane jest przez zarząd z inicjatywy własnej lub na wniosek:
+Walne Zebranie Członków w trybie nadzwyczajnym zwoływane jest przez Zarząd Stowarzyszenia w dowolnym terminie z inicjatywy własnej lub na wniosek:
 
 a) Komisji Rewizyjnej,
```

Comment:
@kpc21 to jest i albo lub? może warto doprecyzować, pisząc "jednej z poniższych grup"

Reply:
Zmieniłem po prostu na "lub", wygląda brzydziej, ale myślę, że czytelniej się nie da.